tox.ini

```diff
@@ -140,6 +141,10 @@ commands =
 
 [testenv:flake8-examples]
 basepython = {[testenv:flake8]basepython}
+# This does not actually ignore errors,
+#  it just runs all commands regardless of whether any fail.
+#  If any fail, the final result is still a fail.
```

Comment:
np: extra whitespace?

np: `the final result is still a fail` -> `the final result will still fail`